hackathon_site/event/test_api.py

```diff
@@ -613,3 +614,91 @@ def test_team_get_has_permissions(self):
         data = response.json()
 
         self.assertEqual(expected_response[0], data)
+
+
+class TeamIncidentListViewPostTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.team = Team.objects.create()
+        self.order = Order.objects.create(status="Cart", team=self.team)
+
+        self.hardware = Hardware.objects.create(
+            name="name",
+            model_number="model",
+            manufacturer="manufacturer",
+            datasheet="/datasheet/location/",
+            notes="notes",
+            quantity_available=4,
+            max_per_team=1,
+            picture="/picture/location",
+        )
+        self.order_item = OrderItem.objects.create(
+            order=self.order, hardware=self.hardware,
+        )
+
+        self.request_data = {
+            "state": "Broken",
+            "time_occurred": "2022-08-08T01:18:00-04:00",
+            "description": "Description",
+            "order_item": self.order_item.id,
+            "team_id": 1,
```

Comment:
This shouldn't need to be in the request, it's a method field on the serializer and not actually going into the model.

If itt throws an error without it, can you add `team_id` to the read only fields for the serializer? Might as well add `created_at` and `update_at` to the read only fields too, since they are set automatically.

Reply:
yep done

Reply:
Still needs to be removed, not a required part of the request